src/engine/strat_engine/cmd.rs

```diff
@@ -61,14 +98,11 @@ lazy_static! {
         (UDEVADM.to_string(), find_binary(UDEVADM)),
         (XFS_DB.to_string(), find_binary(XFS_DB)),
         (XFS_GROWFS.to_string(), find_binary(XFS_GROWFS)),
-        (CLEVIS.to_string(), find_binary(CLEVIS)),
-        (CLEVIS_BIND.to_string(), find_binary(CLEVIS_BIND)),
-        (CLEVIS_UNBIND.to_string(), find_binary(CLEVIS_UNBIND)),
-        (CLEVIS_UNLOCK.to_string(), find_binary(CLEVIS_UNLOCK)),
     ]
     .iter()
     .cloned()
     .collect();
+    static ref USE_CLEVIS: bool = CLEVIS_EXECS.iter().all(|exec| find_binary(exec).is_some());
```

Comment:
There's something we overlooked when we first added the clevis commands: for all the other commands we use the absolute path that is found by ```find_binary``` to invoke the binary. I believe that the reason was that the ```$PATH``` environment variable had been found to be missing in certain situations. We should have this solution do that also.